lib/middlewares/reject_invalid_request_uri.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
If these changes look familiar, it's because they were already approved in https://github.com/GSA/search-gov/pull/736/files#diff-1b78e73a5db3338d94d2e1b5d61435284fab90dd3e1c67279d66a9f7e824d727. Adding the magic comment to request/controller files & specs always broke this, to it will be good to get it into master sooner rather than later.